packages/marko-web-omeda-identity-x/index.js

```diff
@@ -93,6 +96,9 @@ module.exports = (app, params = {}) => {
     omedaPromoCodeCookieName: Joi.string().default('omeda_promo_code'),
     omedaPromoCodeDefault: Joi.string(),
     omedaRapidIdentifyProp: Joi.string().default('$omedaRapidIdentify'),
+    onLoginLinkSentFormatter: Joi.function(),
+    onAuthenticationSuccessFormatter: Joi.function(),
+    onUserProfileUpdateFormatter: Joi.function(),
```

Comment:
I believe these _should_ be required still. If someone explicitly passes `null` for `onLoginLinkSentFormatter`, this will fatal. Test this to make sure

Reply:
@solocommand I don't believe it needs to be required here as it is setting the default value here and when it runs validation on the individual hooks using these it is doing a .function().required() check.  I believe at this step it need to set the default value or validate that what is being set is a function.  This would insure that at the second place this is used it wold have a function set at the key in the object.   If I am miss understanding let me know.  But as soon as i make it required at this point it doesn't function as I would expect for setting the default value.  Meaning if you set .required() it enforce that it be present so there is no point in setting the default value as you have to set it when you call the initial call.